neard/res/mainnet_genesis.json

```diff
@@ -0,0 +1,336 @@
+        "keccak256_byte": 5536829,
+        "keccak512_base": 1420185260,
+        "keccak512_byte": 11073658,
+        "log_base": 0,
+        "log_byte": 0,
+        "storage_write_base": 21058769282,
+        "storage_write_key_byte": 23447086,
+        "storage_write_value_byte": 9437547,
+        "storage_write_evicted_byte": 0,
+        "storage_read_base": 19352220621,
+        "storage_read_key_byte": 4792496,
+        "storage_read_value_byte": 139743,
+        "storage_remove_base": 109578968621,
+        "storage_remove_key_byte": 9512022,
+        "storage_remove_ret_value_byte": 0,
+        "storage_has_key_base": 20019912030,
+        "storage_has_key_byte": 4647597,
+        "storage_iter_create_prefix_base": 28443562030,
+        "storage_iter_create_prefix_byte": 442354,
+        "storage_iter_create_range_base": 25804628282,
+        "storage_iter_create_from_byte": 429608,
+        "storage_iter_create_to_byte": 1302886,
+        "storage_iter_next_base": 24213271567,
+        "storage_iter_next_key_byte": 0,
+        "storage_iter_next_value_byte": 1343211668,
+        "touching_trie_node": 1,
+        "promise_and_base": 0,
+        "promise_and_per_promise": 672136,
+        "promise_return": 34854215
+      },
+      "grow_mem_cost": 1,
+      "regular_op_cost": 3856371,
+      "limit_config": {
+        "max_gas_burnt": 200000000000000,
+        "max_gas_burnt_view": 200000000000000,
+        "max_stack_height": 16384,
+        "initial_memory_pages": 1024,
+        "max_memory_pages": 2048,
+        "registers_memory_limit": 1073741824,
+        "max_register_size": 104857600,
+        "max_number_registers": 100,
+        "max_number_logs": 100,
+        "max_total_log_length": 16384,
+        "max_total_prepaid_gas": 10000000000000000,
+        "max_actions_per_receipt": 100,
+        "max_number_bytes_method_names": 2000,
+        "max_length_method_name": 256,
+        "max_arguments_length": 4194304,
+        "max_length_returned_data": 4194304,
+        "max_contract_size": 4194304,
+        "max_length_storage_key": 4194304,
+        "max_length_storage_value": 4194304,
+        "max_promises_per_function_call_action": 1024,
+        "max_number_input_data_dependencies": 128
+      }
+    },
+    "account_creation_config": {
+      "min_allowed_top_level_account_length": 32,
+      "registrar_account_id": "registrar"
+    }
+  },
+  "validators": [
+    {
+      "account_id": "nfval1.near",
+      "public_key": "ed25519:9BmAFNRTa5mRRXpSAm6MxSEeqRASDGNh2FuuwZ4gyxTw",
+      "amount": "50000000000000000000000000000"
+    }
+  ],
+  "transaction_validity_period": 21600,
+  "developer_reward_percentage": 30,
+  "protocol_reward_percentage": 10,
+  "max_inflation_rate": 0,
+  "total_supply": "1000000000000000000000000000000000",
+  "num_blocks_per_year": 31536000,
+  "protocol_treasury_account": "treasury.near",
+  "fishermen_threshold": "340282366920938463463374607431768211455",
+  "records": [
```

Comment:
Why should we separate `near`, `treasury.near`, `contributors.near`, and `foundation.near` accounts?

Reply:
See referenced issues.